news/_posts/2020-03-20-ps17-patch-release-lifecycle.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: "PrestaShop 1.7 patch release lifecycle"
+subtitle: "The journey of a patch release of PS 1.7, step by step"
+date:   2020-03-20 16:00:00
+authors: [ MathieuFerment ]
+icon: icon-code
+tags:
+ - releases
+ - 1.7
+ - beta
+---
+
+
+Following previous post, here is more explanations about software lifecycle at PrestaShop, focusing on patch releases and the release process.
```

Comment:
please add the title of the previous article, and on it, a link to it,
something like

'Following previous post about [Minor Release Lifecycle](https://build.prestashop.com/news/ps17-minor-release-lifecycle/), ... '